components/panel/src/PanelBody.js

```diff
@@ -1,24 +1,14 @@
 /** @jsx jsx */
 
 import React from 'react';
-import { jsx, useTheme, paint } from '@westpac/core';
-import { propTypes, defaultProps } from './Panel';
+import { jsx, useMediaQuery } from '@westpac/core';
 
 // ==============================
 // Component
 // ==============================
 
-export const PanelBody = ({ appearance, ...props }) => {
-	const { breakpoints, panel } = useTheme();
-	const mq = paint(breakpoints);
+export const PanelBody = props => {
+	const mq = useMediaQuery(breakpoints);
```

Comment:
We're not passing `breakpoints` anymore here, they're provided in the `useMediaQuery` hook.

```suggestion
      const mq = useMediaQuery();
```

Reply:
My mistake. Was late 😬